catalyst/utils/config.py

```diff
@@ -254,6 +254,15 @@ def parse_config_args(*, config, args, unknown_args):
                 continue
             config["args"][key] = value
 
+    if "autoresume" in config["args"] and \
+            "logdir" in config["args"] and \
+            "resume" not in config["args"]:
+        log_dir = config["args"]["logdir"]
+        checkpoint_filename = os.path.join(
+            log_dir, "checkpoints", "last_full.pth"
+        )
+        if os.path.isfile(checkpoint_filename):
+            config["resume"] = checkpoint_filename
```

Comment:
does this feature supports loading checkpoint for Stage2/etc? the case is the following
1. you are training multi-stage pipeline on slurm
2. stage1 goes okay
3. stage2 breaks
4. you are loading the checkpoint from stage2 to resume the training

which stage would it become training?

Reply:
Slurm is not important in this case. I stopped training on stage1 and stage2, and it resumed fine with pull request 629. This pull request is more like syntactic sugar.